README.md

```diff
@@ -1,29 +1,28 @@
 # Tesseract OCR
 
 [![Build status](https://ci.appveyor.com/api/projects/status/miah0ikfsf0j3819/branch/master?svg=true)](https://ci.appveyor.com/project/zdenop/tesseract/)
-[![Build status](https://github.com/tesseract-ocr/tesseract/workflows/sw/badge.svg)](https://github.com/tesseract-ocr/tesseract/actions/workflows/sw.yml)<br>
+[![Build status](https://github.com/tesseract-ocr/tesseract/workflows/sw/badge.svg)](https://github.com/tesseract-ocr/tesseract/actions/workflows/sw.yml)
+
```

Comment:
This changes the layout:
`<br>` or `<br/>` below only cause a line break, while the new empty line starts a new paragraph.

Reply:
Using `\` instead of `<br>` at the line ending would preserve the layout.
See [GitHub markdown documention](https://github.github.com/gfm/#hard-line-breaks).